src/script/components/asset/controls/seekBar.js

```diff
@@ -118,15 +115,15 @@ z.components.SeekBarComponent = class SeekBarComponent {
     this.media_element.removeEventListener('timeupdate', this.on_timeupdate);
     this.media_element.removeEventListener('ended', this.on_ended);
   }
-};
+}
 
 ko.components.register('seek-bar', {
   template: `
     <input type="range" value="0" max="100" data-bind="css: {'show-seek-bar-thumb': show_seek_bar_thumb, 'element-disabled': disabled}">
   `,
   viewModel: {
     createViewModel(params, component_info) {
-      return new z.components.SeekBarComponent(params, component_info);
+      return new SeekBarComponent(params, component_info);
```

Comment:
Maybe it's possible to replace `createViewModel ... return new SeekBarComponent` with just `SeekBarComponent`?

Reply:
Unfortunately it is not that easy, as the `componentInfo` is only passed to the `createViewModel` method